core/renderers/common/info.js

```diff
@@ -138,6 +138,13 @@ Blockly.blockRendering.RenderInfo = function(renderer, block) {
    */
   this.rows = [];
 
+  /**
+   * The number of active rows on a block.
```

Comment:
Update the comment to say that while creating rows, it' the currently active one, and after creating rows it's the total number of rows on the block.

Reply:
I think instead I'm going to call it inputRowCount as I'm only interested in the number of input rows.

Reply:
I'm not sure about that comment. It's not the currently active one, it's not even the index of the row of the active row. It's the current count of input rows created.

Reply:
How does ``The total number of input rows added onto the block.`` sound?

Reply:
Sounds good. I thought you were using the value to determine the negative padding for each row as you went, but you're only using whether it's greater than 1.